spyro/receivers/Receivers.py

```diff
@@ -347,7 +333,21 @@ def __func_build_cell_tabulations_3D(self):
 
         return cell_tabulations
 
-
+    def setPointCloudRec(self, comm, paralel_z=True):
```

Comment:
Can you follow the convention of snake case? https://www.google.com/search?client=safari&rls=en&q=snake+case&ie=UTF-8&oe=UTF-8

Reply:
Ok. I'm modifying here.

Reply:
done!